include/alpaka/dev/DevCudaRt.hpp

```diff
@@ -93,6 +93,7 @@ namespace alpaka
                     dev::DevCudaRt const & dev)
                 -> std::string
                 {
+                    // There is cudaDeviceGetAttribute as faster alternative to cudaGetDeviceProperties to get a single device property but it has no option to get the name
```

Comment:
Maybe just use `cuDeviceGetName`?

Reply:
`cuDeviceGetName` is driver API. Currently, we are using only in one of the alpaka tests the driver API.
Since I do not think that someone will call this function within time-critical code we should not use `cuDeviceGetName` even if it looks like an alternative.

Note: if you use the driver API you must also take care because the device order is not required equal to the runtime api.